src/routes/VideoPage.jsx

```diff
@@ -66,7 +79,14 @@ const VideoPage = () => {
             </div>
             <div className='video-actions'>
               <RequiresAuth>
-                <i className='action-icon fa-regular fa-clock fa-lg'></i>
+                <i
+                  className='action-icon fa-regular fa-clock fa-lg'
+                  style={{ opacity: `${isInWatchlater ? "100%" : "50%"}` }}
```

Comment:
You can rather set opacity as 1 and 0.5,  as I think it is not supported in all browsers in percentage

Reply:
Really appreciate the reviews, will definitely solve the issues in a later commit